CHANGES.md

```diff
@@ -1,3 +1,12 @@
+## v0.2.1 (2017-06-15)
+
+* Use [@@noalloc] rather than "noalloc"
```

Comment:
Good point. I'll bump to 4.03 (which should be fine as that's our current Mirage minimum?)